client/components/domains/map-domain-step/index.jsx

```diff
@@ -111,14 +111,17 @@ var MapDomainStep = React.createClass( {
 
 		return (
 			<div className="domain-search-results__domain-availability is-mapping-suggestion">
-				<div className="domain-search-results__domain-availability-copy notice is-success">
+				<Notice
+					className="domain-search-results__domain-availability-copy"
```

Comment:
Actually, from what I can see, this style doesn't even exist - so it can be removed from this notice (as well as the one in `client/components/domains/domain-search-results/index.jsx`).